tests/foreman/ui/test_csv_export.py

```diff
@@ -0,0 +1,781 @@
+        """
+
+    @tier1
+    @stubbed()
+    def test_positive_location_filter(self):
+        """
+        :id: d526794c-07a2-4db2-a737-01c2827d5cec
+
+        :setup:
+            1. Create hosts in locA and locB
+
+        :steps:
+            1. Open host pages for locA
+            2. Export Host page to CSV
+            3. Open host page for locB
+            4. Export Host page to CSV
+
+
+        :expectedresults:
+            Only Org A hosts are show in locA CSV
+            and only locB hosts are show in locB csv.
+        """
+
+    @tier1
+    @stubbed()
+    def test_negative_location_filter(self):
+        """
+        :id: 0d6da07f-b5df-49a9-816d-033daa504482
+        :setup:
+            1. Create hosts in locA and locB
+
+        :steps:
+            1. Set Location to "Any"
+            2. Export Host page to CSV.
+
+        :expectedresults: All hosts are listed in CSV file.
+        """
+
+    @tier1
+    @stubbed()
+    def test_combo_filter(self):
+        """
+        :id: 46b61542-994a-48ca-b4b0-90c886158614
+
+        :setup:
+            1. Chose some host filter (filterA)
+            2. Create multiple hosts, such that some match
+               filterA and some dont
+
+        :steps:
+            1. Filter the host page with filterA
+            2. Export the host page to CSV
+
+        :expectedresults: CSV only contains filtered hosts
+        """
+
+    @tier1
+    @stubbed()
+    def test_positive_filter_multi_page(self):
+        """
+        :id: c506d83f-de27-43cd-840c-dae57b889a8f
+
+        :setup:
+            1. Chose some host filter (filterA)
+            2. Create greater then per page max of hosts, such that
+               some match filterA and some dont
+        :steps:
+            1. Filter the host page with filterA
+            2. Export the host page to CSV
+
+        :expectedresults: CSV only contains filtered hosts from all pages
+        """
+
+    @tier1
+    @stubbed()
+    def test_positive_filter_single_page(self):
+        """
+
+        :id: 67f0bf54-0db9-45f4-899b-081ed1c83e70
+
+        :setup:
+            1. Chose some host filter (filterA)
+            2. Create less then per page max of hosts, such that
+               some match filterA and some dont
+        :steps:
+            1. Filter the host page with filterA
+            2. Export the host page to CSV
+
+        :expectedresults: CSV contains filtered hosts from page.
+        """
+
+
+class UICSVExportHostsColumnsTestCase(UITestCase):
+
+    @tier1
+    @stubbed()
+    def test_positive_column_contents(self):
+        """
+        :id: 2f86176d-f90a-4589-ab52-a56fbd5fc713
+
+        :setup:
+            Populate hosts multiple hosts with a mix of Puppet Enviromets,
+            Hostgroups, OS, etc
+
+        :steps:
+            1. Export the CSV Page in the UI
+
+        :expectedresults:
+            Exported CSV will contain text representations of all columns.
+
+
```

Comment:
Too much blank lines at the end of docstring

Reply:
@rochacbruno @sghai if you think its ok as is please merge it.